CONTRIBUTING.rst

```diff
@@ -117,8 +307,9 @@ Core Contributors are developers that actively work and maintain the repository.
 They are the only ones who accept pull requests and push commits directly to
 the **pysteps** repository.
 
-To include the contributions for collaborators, we use the usual GitHub
-pull-request flow. In their simplest form, pull requests are a mechanism for
+To include the contributions for collaborators, we use the usual
+`GitHub pull-request flow <https://help.github.com/en/articles/github-flow>`_.
+In their simplest form, pull requests are a mechanism for
 a collaborator to notify to the pysteps project about completed a feature.
```

Comment:
"about a completed feature"

Reply:
Change made in 6f9b0405c4b4fce428bc92274a11381583108a4a